app/routes/screens/racePredictor/components/locationMapMarker/LocationMapMarker.js

```diff
@@ -0,0 +1,126 @@
+import React, { PropTypes } from "react";
+import { Text, View, PermissionsAndroid, Platform } from "react-native";
+import MapView from "react-native-maps";
+import isEqual from "lodash/isEqual";
+
+import styles from "./styles";
+
+const GEOLOCATION_OPTIONS = {
+  enableHighAccuracy: true,
+  timeout: 20000,
+  maximumAge: 1000
+};
+const ANCHOR = { x: 0.5, y: 0.5 };
+
+const propTypes = {
+  ...MapView.Marker.propTypes,
+  // override this prop to make it optional
+  coordinate: PropTypes.shape({
+    latitude: PropTypes.number.isRequired,
+    longitude: PropTypes.number.isRequired
+  }),
+  children: PropTypes.node,
+  geolocationOptions: PropTypes.shape({
+    enableHighAccuracy: PropTypes.bool,
+    timeout: PropTypes.number,
+    maximumAge: PropTypes.number
+  }),
+  heading: PropTypes.number,
+  enableHack: PropTypes.bool
+};
+
+const defaultProps = {
+  enableHack: false,
+  geolocationOptions: GEOLOCATION_OPTIONS,
+  heading: 0,
+  children: null,
+  coordinate: null
+};
+
+export default class MyLocationMapMarker extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.mounted = false;
```

Comment:
As I told you, AirBnb hack!! :) 
This is going to be removed ASAP reloaddux (aka redux-cargo or whatever) will be implemented.